README.md

```diff
@@ -114,8 +114,10 @@ This mode is used to find the maximum and throughput that can be obtained from t
 This mode can be used for both producers and consumers.
 
 ```
```

Comment:
The example text only should go within the triple back-ticks. Your change puts the "For example" header in the triple back-ticks. See the rendered file in Github.

Reply:
Looking at the rendered file I don't see any `For Example` headers inside the formatted text blocks.  The reason I didn't put the command line example inside a pre-formatted text block is because then it appears as one line in a scrollable box that is difficult to read.

Reply:
The formatting is still messed up but I now see that it is not a result of this PR.
For reference, I usually format split long command lines with a backslash separator that can be copied directly to a terminal. For instance:
```
cp /my/long/file1 \
/my/long/file2
```

Reply:
Yes, I was going to reformat it like that but didn't want to detract from the examples.

BTW, I don't have write access so I can't merge this.

Reply:
@maddisondavid I will merge this PR (in our committer guidelines, it is encouraged that a person different from the author of the PR merges the code).